src/MIDebugEngine/ResourceStrings.resx

```diff
@@ -213,7 +213,9 @@
   <data name="Error_ExePathInvalid" xml:space="preserve">
     <value>Program path '{0}' is missing or invalid.
 
-{1} failed with message: {2}</value>
+{1} failed with message: {2}
+
+Updates can cause binaries replaced on disk and resulting in this failure. Try re-launching the application or restarting the machine.</value>
```

Comment:
nit: The first sentence seems a little awkward to me. How about:

This may occur if the process's executable was changed after the process was started, such as when installing an update.